.github/workflows/build.yml

```diff
@@ -10,14 +10,20 @@ jobs:
           - ruby: 3.3
             gemfile: Gemfile
           - ruby: 3.2
-            gemfile: Gemfile
+            gemfile: gemfiles/rails71.gemfile
           - ruby: 3.1
             gemfile: gemfiles/rails70.gemfile
           - ruby: "3.0"
             gemfile: gemfiles/rails61.gemfile
+          - ruby: 3.3
+            gemfile: gemfiles/mongoid9.gemfile
+            mongodb: true
           - ruby: 3.1
             gemfile: gemfiles/mongoid8.gemfile
             mongodb: true
+          - ruby: 3.1
+            gemfile: gemfiles/mongoid9.gemfile
+            mongodb: true
```

Comment:
I added this check myself, since it's the main reason why we're updating this to begin with.